include/oneapi/dpl/pstl/numeric_impl.h

```diff
@@ -217,7 +212,8 @@ __brick_transform_scan(_RandomAccessIterator __first, _RandomAccessIterator __la
 
 template <class _RandomAccessIterator, class _OutputIterator, class _UnaryOperation, class _Tp, class _BinaryOperation,
           class _Inclusive>
-typename ::std::enable_if<is_arithmetic_udop<_Tp, _BinaryOperation>::value, ::std::pair<_OutputIterator, _Tp>>::type
+typename ::std::enable_if<!oneapi::dpl::__unseq_backend::is_arithmetic_plus<_Tp, _BinaryOperation>::value,
```

Comment:
For other case, when we are dispatching to UDS code and that code requires creating a identity object by calling a default constrictor, we should keep such behavior by performance reasons. And we have the documented such restriction in the "Known Issues and Limitations" section of the release notes document.
If a user cannot provide such default constructor for his type, he can use just `par `or `seq` policy instead of `par_unseq `or `unseq`.

Reply:
I'm not finding this restriction in the "Known Issues and Limitations" section or release notes currently.  Are you saying that we should add such a restriction to the release notes or the guide?